server/middleware/contest.js

```diff
@@ -0,0 +1,18 @@
+const User = require("../models/User");
+module.exports.protect = async (req, res, next) => {
```

Comment:
You don't need to create this, please see `auth.js` in the middleware folder, use that instead